docs/generate-devsite-utilities.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+
+source ../toolversions.sh
+
+install_docfx
+
+if [[ -z "$1" || -z "$2" ]]
+then
+  echo 'Please specify utility and version'
+  echo 'Known types: google-apis protobuf common-proto grpc gax'
+  exit 1
+fi
+
+if [[ "$3" != "" ]]
+then
+  declare -r SERVICE_ACCOUNT_JSON=$3
+else
+  declare -r SERVICE_ACCOUNT_JSON=""
+fi
+
+set -eu -o pipefail
+
+declare -r DEVSITE_STAGING_BUCKET=docs-staging-v2
+declare -r VERSION=$2
+
+# These are listed in the order of dependencies:
+# each type only depends on types listed earlier
+case $1 in
+"protobuf")
```

Comment:
Is there a `(` missing here? And all the rest of the cases.

Reply:
Nope, that's just the syntax for bash cases

Reply:
Having said that, the indentation is a bit skew. Will fix that.